bower.json

```diff
@@ -5,24 +5,31 @@
   "dependencies": {
     "underscore": "~1.6.0",
     "jquery": "~1.11",
-    "bootstrap": "~3.0",
+    "bootstrap": "~3.3",
     "angular": "~1.2",
     "angular-route": "~1.2",
     "respond": "~1.4",
     "sjcl": "~1.0",
     "html5shiv": "~3.6",
-    "angular-bootstrap": "~0.11.0",
-    "angular-ui-bootstrap": "~0.11.0",
-    "d3": "~3.4.2",
+    "angular-bootstrap": "https://github.com/Nevtep/bootstrap-bower.git#master",
+    "angular-ui-bootstrap": "~0.12.0",
     "jsrsasign": "~4.2.1",
     "FileSaver": null,
     "big.js": "x.x.x",
     "forge": "https://github.com/digitalbazaar/forge.git#0.2.30",
+    "angular-password-strength": "https://github.com/curtislacy/angular-password-strength.git#Caret-Operator",
+    "angular-timer": "~1.1.6",
+    "ngInfiniteScroll": "1.0.0",
     "numeraljs": "https://github.com/adamwdraper/Numeral-js.git#1.5.3",
     "angular-numeraljs": "https://github.com/sagyeli/angular-numeraljs.git#master",
-    "angular-password-strength": "https://github.com/curtislacy/angular-password-strength.git#Caret-Operator",
-    "angular-idle": "https://github.com/hackedbychinese/ng-idle.git#0.3.5",
+    "ng-idle": "https://github.com/hackedbychinese/ng-idle.git#0.3.5",
     "angular-re-captcha": "~0.0.3",
```

Comment:
@Nevtep   i've forged all these individual user repos to the omniwallet repo. 
Can we please go ahead an update all the direct github links to use the projects as they are forked on https://github.com/Omniwallet